kassy

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# kassy - A bash CLI tool for running kassy in docker
+# Written By: Matthew Hartstonge <matt@mykro.co.nz>
+
+# Soak in user params
+ARGS="$@"
+
+function usage() {
+  echo "usage: kassy [--all] [--fb] [--skype] [--slack] [--test]               "
```

Comment:
The only thing that is going to be annoying about this is updating this file every time an integration module is added. Although I'm not sure what can be done about that?